### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by anilmurty** [akash-network/community#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by feiveslok** [comment]
this video helping to understand Akash for community https://www.tiktok.com/@danielonchain/video/7247897240319118610

**Comment by Moekali84** [comment]
Hello,

is there a way to run something like Webodm on the Akash network. the problem with this is that it normally needs Docker Compose to run and the multiple scripts within the application. https://hub.docker.com/r/opendronemap/nodeodm

**Reply by anilmurty** [reply]
It sounds like a normal docker image - don't think there should be an issue running it. Have you tried running a docker image on Akash? There are a bunch of examples here (https://github.com/akash-network/awesome-akash) and you can always ask for help/ support in our discord server

**Comment by Moekali84** [comment]
Question! why every time i add my IP address to the expose section of the code. i dont get any bids? everything was working from two days ago. now the same code that i past into the builder does not yield any bidders unless i remove the expose section from the code. any Help is greatly appreciated.

**Comment by RizaZare** [comment]
Question- How can I find some technical info on Akash Network like TPS(transaction per second), TTF and etc?

**Comment by KaylemJ** [comment]
Refreshed welcome message can be found here: https://github.com/orgs/akash-network/discussions/1355 
Joe Deng, Akash Community Manager
